src/Neo.Compiler.CSharp/Optimizer/DumpNef.cs

```diff
@@ -292,6 +300,10 @@ public static string GenerateDumpNef(NefFile nef, JToken? debugInfo)
                 }
             }
 
+            if (debugInfo == null && manifest != null)
```

Comment:
```suggestion
            if (manifest != null)
```
Manifest is more trustable, isn't it?

Reply:
> Manifest is more trustable, isn't it?

Yes it is.